### PR TITLE
[DS-75] Added hopper components to chromatic stories

### PR DESCRIPTION
**Pull request by vicky-comeau**
## Summary

We needed to integrate Hopper components with existing Orbiter components.

## What I did

- Removed unused hopper chromatic stories
- Added hopper components to Orbiter component chromatic tests

## How to test

- Is this testable with Jest or Chromatic screenshots? yes, chromatic screenshots

### CODE REVIEW

**Comment by victortrinh2** [comment]
Changes are good, just need to check if the small changes on chromatic are good or if we need to deal with them.

**Review by victortrinh2 — DISMISSED** [review]

File: .changeset/shaggy-pens-rescue.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (alexasselin008) [inline .changeset/shaggy-pens-rescue.md:1]: This is not what the release is for. Consumers don't care about chromatic stories. 
In general you wouldn't need a release for this. But i'm guessing you are releasing so that the fixes in the code such as dialog and index.css are released. So list those changes instead

File: packages/components/src/alert/tests/chromatic/AlertTrigger-hopper.stories.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
+import { Content } from "@components/placeholders";
 import { Alert, AlertTrigger } from "@components/alert";
+import { Heading } from "@hopper-ui/components";
 import { Button } from "@components/button";
```
Comment (alexasselin008) [inline packages/components/src/alert/tests/chromatic/AlertTrigger-hopper.stories.tsx:4]: why not the button?

File: packages/components/src/alert/tests/chromatic/createTestSuite-hopper.jsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Content } from "@components/placeholders";
 import { Div } from "@components/html";
```
Comment (alexasselin008) [inline packages/components/src/alert/tests/chromatic/createTestSuite-hopper.jsx:2]: why not deplace the html element as well?

File: packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
+import { Content } from "@components/placeholders";
 import { Field, Label } from "@components/field";
 
 import { Div } from "@components/html";
 import { Form } from "@components/form";
-import { Heading } from "@components/typography";
 import { Image } from "@components/image";
 import { Launch } from "./assets";
 import { Popover } from "@components/popover";
 import { TextInput } from "@components/text-input";
```
Comment (alexasselin008) [inline packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx:9]: why not the text input?

File: packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx [hunk 5]
```diff
@@ -1,15 +1,14 @@
-import { Button, ButtonGroup } from "@components/button";
-import { Content, Footer } from "@components/placeholders";
+import { Content } from "@components/placeholders";
 import { Field, Label } from "@components/field";
 
 import { Div } from "@components/html";
 import { Form } from "@components/form";
```
Comment (alexasselin008) [inline packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx:5]: why not the form?

File: packages/components/src/alert/tests/chromatic/createTestSuite-hopper.jsx (hunk 3 above)
Comment (vicky-comeau) [reply packages/components/src/alert/tests/chromatic/createTestSuite-hopper.jsx:2]: hopper's Div doesn't seem to work in Orbiter. I created a bug for this.

File: packages/components/src/alert/tests/chromatic/AlertTrigger-hopper.stories.tsx (hunk 2 above)
Comment (vicky-comeau) [reply packages/components/src/alert/tests/chromatic/AlertTrigger-hopper.stories.tsx:4]: I may have missed this one, thanks

File: packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx (hunk 4 above)
Comment (vicky-comeau) [reply packages/components/src/popover/tests/chromatic/Popover-hopper.stories.tsx:9]: I could try replacing this one. I kept wondering if we only wanted to replace ones that were slots or not.

File: packages/components/src/tag/tests/chromatic/TagList-hopper.stories.tsx [hunk 6]
```diff
@@ -115,6 +115,7 @@ export const WithClearButtonEmpty: TagListStory = {
     storyName: "with clear button & empty",
     render: () => (
         <TagList onClear={() => {}}>
```
Comment (alexasselin008) [inline packages/components/src/tag/tests/chromatic/TagList-hopper.stories.tsx:117]: why is this necessary? what was the error here?

**Review by alexasselin008 — APPROVED** [review]
